lib/phobos_db_checkpoint/handler.rb

```diff
@@ -31,7 +45,15 @@ def around_consume(payload, metadata)
           end
 
           event_action = instrument('db_checkpoint.event_action', event_metadata) do
-            yield
+            begin
```

Comment:
ok. good point about dbCheckpoint being a plugin.

@klippx , do you also agree? do we have a verdict =P?